app-libs/stf/src/test_genesis.rs

```diff
@@ -67,9 +73,23 @@ pub fn test_genesis_setup(state: &mut SgxExternalities) {
 		(ALICE_ENCODED.into(), ALICE_FUNDS, ALICE_FUNDS),
 	];
 
+	fund_alice_evm_account(&mut endowees);
```

Comment:
I think you can add the `#[cfg(feature = "evm")]` feature flag directly above this function call, then you don't need to define this function at all in the not-evm feature.

Reply:
ok!

Reply:
changing the functions signature will solve this problem